.github/workflows/docker.yml

```diff
@@ -14,7 +14,8 @@ jobs:
     outputs:
       DOCKERHUB_USERNAME: ${{ steps.dockerhub_username.outputs.is_set }}
       DOCKERHUB_TOKEN: ${{ steps.dockerhub_token.outputs.is_set }}
-      ALL: ${{ steps.dockerhub_token.outputs.is_set && steps.dockerhub_token.outputs.is_set }}
+      DOCKER_BASE_REPO: ${{ steps.docker_base_repo.outputs.is_set }}
+      ALL: ${{ steps.dockerhub_username.outputs.is_set && steps.dockerhub_token.outputs.is_set && steps.docker_base_repo.outputs.is_set }}
```

Comment:
I'd remove this part and the subsequent checks / variables and just let the push to Dockerhub fail

Reply:
I introduced the checks on secrets/variables in order to verify the image build at every commit/merge, even if secrets were not defined (e.g. contributors without Docker Hub account). In my view, the workflow should fail in case of build error or wrong credentials.

Reply:
The last step and the whole workflow with fail anyway with no credentials, am I missing something?

Reply:
Are you referring to this one?

https://github.com/italia/spid-compliant-certificates-python/blob/d419c7b1b18ec228c6942779c1d6b35d0869470b/.github/workflows/docker.yml#L76-L85

If so, the check on secrets is used to enable/disable the push on the registry (see line 83). In particular, if the event is not a pull request and all the credentials are set, the currently built image must be pushed. If the result of the logical AND is false, the step just builds the image.

Reply:
@bfabio any news about that? What about `DOCKER_BASE_REPO` secret?

Reply:
@psmiraglia `DOCKER_BASE_REPO` is set